scene/gui/line_edit.cpp

```diff
@@ -2523,6 +2566,31 @@ Ref<Texture2D> LineEdit::get_right_icon() {
 	return right_icon;
 }
 
+void LineEdit::set_icon_expand_mode(ExpandMode p_mode) {
+	if (icon_expand_mode != p_mode) {
```

Comment:
```suggestion
	if (icon_expand_mode = p_mode) {
		return;
	}
```
Early returns are preferred. Same below.